webpack.config.js

```diff
@@ -457,17 +527,21 @@ module.exports = (env, argv) => {
         // configuration for the webpack-dev-server
         devServer: {
             // serve unwebpacked assets from webapp.
-            contentBase: './webapp',
+            contentBase: [
```

Comment:
again, comment could with updating here - I don't really understand what's going on

Reply:
It was basically misconfigured, `contentBase` is required for dev server to properly understand where it should search for files.
I'll figure out if we can make it a bit smaller, but basically it was too smalll.

Reply:
Looks like I was wrong, thanks for catching it! Seems that it was just a leftover of my investigations.